jwst/transforms/models.py

```diff
@@ -57,6 +57,31 @@ class GrismObject(namedtuple('GrismObject', ("sid",
                                              ), rename=False)):
     """ Grism Objects identified from a direct image catalog and segment map.
 
+    Parameters
+    ----------
+    sid : int
+        source identifed
+    xcentroid : float
+        x center of object in pixels
+    ycentroid : float
+        y center of object in pixels
+    order_bounding : dict
+        Contains the object x,y bounding location on the image
```

Comment:
Is this a mapping `{order: (x,y)}`?

Reply:
It's a member of the GrismObject, a dictionary keyed on spectral order that contains a tuple of ((ymin, ymax), (xmin, xmax))